src/main/java/com/yelp/nrtsearch/server/luceneserver/IndexState.java

```diff
@@ -824,9 +814,9 @@ public static boolean isChildName(String name) {
    * Live setting: set the mininum refresh time (seconds), which is the longest amount of time a
    * client may wait for a searcher to reopen.
    */
-  public synchronized void setMinRefreshSec(double min) {
```

Comment:
The set methods will still need to be synchronized. Otherwise, there is a race condition and the local class members may have a different value than what is in the json state.